src/http/RegisteredHeadersHash.cci

```diff
@@ -1,35 +1,35 @@
 /* C++ code produced by gperf version 3.1 */
-/* Command-line: gperf -m 100000 RegisteredHeadersHash.gperf  */
+/* Command-line: gperf -m 100000 ./RegisteredHeadersHash.gperf  */
```

Comment:
I don't know why it has generated like this.
I ran `make -C src/http gperf-files` as suggested.

Reply:
I fixed that and reformatted in 42991ff.

Reply:
Thank you.